pkg/oc/cli/startbuild/startbuild.go

```diff
@@ -890,7 +888,7 @@ func gitRefInfo(repo git.Repository, dir, ref string) (buildapi.GitRefInfo, erro
 }
 
 // WaitForBuildComplete waits for a build identified by the name to complete
-func WaitForBuildComplete(c buildclient.BuildResourceInterface, name string) error {
+func WaitForBuildComplete(c buildclientv1.BuildResourceInterface, name string) error {
```

Comment:
the versioned / v1 build client does not actually have a `BuildResourceInterface`